bindings/nodejs/generated.js

```diff
@@ -271,10 +271,11 @@ if (!nativeBinding) {
   throw new Error(`Failed to load native binding`)
 }
 
-const { Operator, Entry, Metadata, Lister, BlockingLister } = nativeBinding
+const { Operator, Entry, Metadata, Lister, BlockingLister, RetryLayer } = nativeBinding
 
 module.exports.Operator = Operator
 module.exports.Entry = Entry
 module.exports.Metadata = Metadata
 module.exports.Lister = Lister
 module.exports.BlockingLister = BlockingLister
+module.exports.RetryLayer = RetryLayer
```

Comment:
We will expose `RetryLayer` directly as `opendal`? Do we need a `opendal.layers`'s namespace?

Reply:
It's ok to group layers in a namespace.